tests/e2e/cypress/cypress/integration/templates/templates-test.spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="Cypress" />
+/*
+
+RERO ILS
+Copyright (C) 2020 RERO
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, version 3 of the License.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+before(function () {
+  cy.fixture('users').then(function (userData) {
+    this.users = userData;
+  })
+  cy.fixture('common').then(function (commonData) {
+    this.common = commonData;
+  })
+  cy.fixture('templates').then(function (templateData) {
+    this.templates = templateData;
+  })
+  cy.server();
+  cy.route({method: 'GET', url:'/api/templates/?q=*'}).as('api_template_search')
+  cy.route({method: 'GET', url:'**/professional/records/document**)'}).as('document_editor')
+  cy.route({method: 'POST', url:'/api/templates/'}).as('api_template_create')
+})
+
+describe('Templates: Create and use template for a document', function() {
+
+  before('Setting defaults', function() {
+    cy.visit('')
+    cy.setLanguageToEnglish()
+  })
+
+  beforeEach('Login as admin and clean templaltes', function() {
+    const template = this.templates.templateA
+    cy.api_logout()
+    cy.api_login(this.users.librarians.spock, this.common.uniquePwd)
+    cy.api_delete_items('templates', 'name:"'+template.name+'"')
+  })
+
+  after('Logout user', function() {
+    cy.api_logout()
+  })
+
+  it('Create a new template and use it in the editor', function() {
+    const template = this.templates.templateA
+
+    cy.visit('/professional/records/documents/new')
+    cy.wait(2000)
```

Comment:
Is it necessary to use wait() ? Normally there is a 2 min timeout for cy.get() command.

Reply:
@jma told me some weeks ago than we can't know when the build of the editor has really finished due to a lot of asynchronous calls. So the `visit` return quickly a HTTP 200 response but the editor is still in construction, adding a `wait` for 2 seconds allows to give me more chance than the form building is completed.

Reply:
Ok